examples/03_making_adjustments.py

```diff
@@ -35,27 +35,27 @@
 # Initialize FLORIS with the given input file via FlorisInterface
 fi = FlorisInterface("inputs/gch.yaml")
 
-# Configure this simulation for visualization using the full flow field grid
-solver_settings = {
-    "type": "flow_field_grid",
-    "flow_field_grid_points": [200,100,7]
-}
-fi.reinitialize(solver_settings=solver_settings)
+# # Configure this simulation for visualization using the full flow field grid
```

Comment:
Should this just be removed? Is it not needed anymore?

Reply:
done!